parkhangs-webapp/src/components/EventForm.js

```diff
@@ -160,16 +174,23 @@ class EventForm extends React.Component {
         const eventDetail = this.eventDetail()
         const isNewEvent = this.isNewEvent()
         const showCalendar = this.showCalendar()
+        const showDatePicker = this.showDatePicker()
 
         return (
             <div className="EventForm">
                 <form className="EventForm">
                     {
-                        showCalendar && <div className="formsection date">
+                        showCalendar && <div className="formsection date"> 
                             <label htmlFor="eventDate">Date:</label>
                             <Calendar id="eventDate" value={eventDate} onChange={this.handleUpdateDate}/>
                         </div>
                     }
+                    {
+                        showDatePicker && <div className="formsection date"> 
```

Comment:
I'm not sure I understand why you added a new and different date picker when there's already one up there^?

Reply:
Where is the other date picker? Maybe it’s bc I wasn’t familiar with the component too but I only saw a daypicker which I wasn’t sure was the same thing or if it was more coupled with the Calendar view

Reply:
yeah daypicker allows the user to pick a day that the event is on, and then there's also a time picker. you just pass in  showCalendar={true} to the component.

Reply:
It doesn't work when I passed it in using the existing showCalendar={true}, have you tested it? I noticed it wasn't used in the ModalEventDetail.js.
I'll leave this in for the time being but if you could take a look after your query parts that would be awesome :)

Reply:
It's not used in modaleventdetail because there is no need for a datepicker there, since you click on a date in order to make a new event. I just tested by setting showDatePicker={true} (see line 86 in EventForm.js) and it worked to show the calendar, however there is a bug that causes it to throw an error since the code that passes it a default date changed and broke it recently. I just had to make a new function and call it in the render function (line 162 `const eventDate = this.dayPickerDateFromat()`

```
dayPickerDateFormat = () => {
        if (this.props.eventDateTime) {
            return new Date(this.props.eventDateTime)
        }
        return null
    }
```

and then it showed up in the event form.

Reply:
Okay, the changes have been made! @cmaija